SPECS-SIGNED/mstflint-driver-signed/mstflint-signed.spec

```
@@ -0,0 +1,32 @@
+%define mstflint_unsigned_name mstflint-kernel
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
+%global install_mod_dir lib/modules/%{kver}/extra/%{name}
+Summary:        Mellanox firmware burning tool
+Name:           %{mstflint_unsigned_name}-signed
+Version:        4.21.0
+Release:        4%{?dist}
+License:        Dual BSD/GPL
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Kernel
+URL:            https://github.com/Mellanox/%{name}
+Source0:        https://github.com/Mellanox/%{name}/releases/download/v%{version}-1/%{name}-%{version}-1.tar.gz
+Requires:       kernel >= 5.15.0 and kernel < 5.16.0
+Requires:       kernel-headers >= 5.15.0 and kernel-headers < 5.16.0
+
+%install
+export INSTALL_MOD_PATH=%{buildroot}
+cd kernel
+install -d $INSTALL_MOD_PATH/%{install_mod_dir}
+cp $PWD/mstflint_access.ko $INSTALL_MOD_PATH/%{install_mod_dir}
```

Just a small clean-up and aligning more to what we tend to gravitate towards in other specs:
```suggestion
install -dm 755 %{buildroot}%{install_mod_dir}
install -m 644 %{SOURCE0} %{buildroot}%{install_mod_dir}
```

The missing `/` in `%{buildroot}%{install_mod_dir}` is by design if we take in the `install_mod_dir` from a separate comment at the top of this spec.

The "signed" specs use pre-built sources as input, thus the removal of `cd kernel` and going straight to `%{SOURCE0}` - that macro holds the absolute path to the first source, which in this case is probably going to be that pre-built and signed kernel module.